ui/admin/tests/e2e/README.md

```diff
@@ -0,0 +1,44 @@
+# Admin UI e2e tests
+
+This test suite tests the Boundary Admin UI in an end-to-end setting. It was designed to be run in a
+variety of environments as long as the appropriate environment variables are set. You can choose to
+spin up your own infrastructure or use [Enos](https://github.com/hashicorp/boundary/tree/main/enos)
+to generate it for you. There is an enos scenario in the `boundary` repository that runs these
+tests.
+
+The test suite uses the [Playwright](https://playwright.dev/) framework.
+
+## Getting Started
```

Comment:
It seems that the CLI is required to be in the path in order to run e2e tests.  Is that documented?

Reply:
That's correct. I'll update the README with that info.